### PR TITLE
Update to jupyter-lsp 0.9.3

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by fcollonval** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
 {% set ext_name = "@krassowski/jupyterlab-lsp" %}
-{% set version = "0.9.2" %}
-{% set build_number = 1 %}
+{% set version = "0.9.3" %}
+{% set build_number = 0 %}
 
-{% set py_spec = ">=3.5" %}
+{% set py_spec = ">=3.6" %}
 {% set lab_spec = ">=2.2,<3.0.0a0" %}
 {% set ext_spec = "@^2.0.0" %}
```
Comment (krassowski) [inline recipe/meta.yaml:8]: 2.1.0?

Comment (fcollonval) [reply recipe/meta.yaml:8]: As far as I understood it is not mandatory. But if you think it is safer, I'll update it too.

**Comment by bollwyvl** [comment]
Looking good, but yea, would bump the extension spec. We don't do much testing against different versions of in-house stuff.

Maybe not on this PR (but now's as good a time as any), we could also start expanding our stable of "other things known to work with this version":
- `jupyter-lsp-latex` with `tectonic`, `texlab`, `checktex`
- `jupyter-lsp-r` with `r-language-server` (anything else key?)

And even `-python` would now be supported by a number of addition plugins, e.g. `pyls-mypy`, `-isort`, `-black`, `-memestra`.... maybe `-python-all`? Unfortunately, a lot of them need client config that isn't easy to package (as it doesn't even persist between runs of the language server).

And one of these days, i'll get a darned robot kernel/lsp set up upstream with all the work folk have been doing!

**Comment by fcollonval** [comment]
> would bump the extension spec

I'll do.

Regarding other variant, it will be safer to do it in another PR.

> Note: I should have used a fork because a branch on this repo got publish directly... (conda-forge is already displaying 0.9.3 - sorry for that).

**Comment by bollwyvl** [comment]
Yerp, that's why it's the first checklist item :blush: .

I _highly_ recommend using an `https` URL for the conda-forge repo, locally and naming it somethign like `conda-forge` or `upstream`, and renaming the personal fork to `origin` so that there is at least time for a pause when it asks for a password (or better still, doesn't work at all because 2FA).

Since it's already shipped, i guess I'll just merge, and we can try again if we want to change anything.

**Pull request merged**